### DISCUSSION TITLE
How to Find the Largest Pair of Adjacent Digits in a Number Array in JavaScript?

### DISCUSSION THREAD

**Discussion by selviast** [community/community#140065] [Programming Help]
### Body

/** 2. Largest Pair

Create a function to find the largest pair (two adjacent numbers with the highest value).

Example 1: Input: 641573 Output: 73

Example 2: Input: 12783456 Output: 83

NOTE:

Use return to display the output.
Provide comments explaining the purpose of the code at important lines. */

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ihwan4rfa** [comment]
Here's a solution to find the largest pair of consecutive digits in a number. The function converts the number to a string and then iterates through it, comparing each pair of consecutive digits and updating the result if a larger pair is found. Here's the implementation:

function pasanganTerbesar(number) {
    let strNumb = number.toString();
    let result = 0;

    for (let i = 0; i < strNumb.length - 1; i++){
        let pasang = parseInt(strNumb[i] + strNumb[i + 1]);
        result < pasang ? result = pasang : result;
    }
    
    return result;
}

const number1 = 641573;
const number2 = 12783456;
console.log(pasanganTerbesar(number1));  // Expected output: 73
console.log(pasanganTerbesar(number2));  // Expected output: 83

This function processes the number efficiently by using string manipulation and checks for the largest consecutive digit pair.

**Comment by ebndev** [comment]
Hi @selviast & @ihwan4rfa, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
